railties/lib/rails/application/configuration.rb

```diff
@@ -285,10 +290,6 @@ def load_defaults(target_version)
           if respond_to?(:action_controller)
             action_controller.allow_deprecated_parameters_hash_equality = false
           end
-
-          if respond_to?(:active_record)
-            active_record.sqlite3_adapter_strict_strings_by_default = true
-          end
```

Comment:
There appeared to be no reason to have this separate, so I consolidated it above.